### ISSUE TITLE
Добавление запросов списком

### ISSUE THREAD

**Issue by Stremuzhevskiy** [kassir-ydirect/project#29]
было бы крайне удобно, если добавлять запросы в скрипт можно было не по одному, а сразу списком. Это можно реализовать?

**Comment by betadog** [comment]
т.е. при редактировании объявления ты хочешь видеть одно большое текстовое поле, а не множество полей?

Может сделать как в самом яндексе:
по умолчанию выводится текст фразы, без поля редактирования. При нажатии на фразу формируется текстовое поле для редактирования + кнопка "сохранить"?

по поводу добавления новых фраз следующая идея:
при нажатии на "+ добавить фразы" формируется большое текстовое поле (далее `ТП`) и кнопка "применить".
В `ТП` вставляется подготовленный текст. 
При нажатии на "применить" `ТП` убирается и формируются тексты фраз взятые из `ТП`. Т.е. некий парсинг на лету. 

Что думаешь?

**Comment by Stremuzhevskiy** [comment]
Да, можно так. единственное, нужно уточнить форматы размещения новых фраз в тп: как их разделять между собой - пробелами? запятыми? Предлагаю пробелами.
На них нужно также наложить фильтрацию по символам: убирать двоеточия и т.п. К слову, вчера в каком то аккаунте (если не ошибаюсь в Сочи), видел что в ключевики попадает двоеточие между словами, выглядит примерно вот так: "текст:текст". Прошу проверить этот момент.
